src/components/buttons/button-styles.ts

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+export const ButtonContainer = styled.div`
+  button {
```

Comment:
Você precisa dessa div em volta do botão? 
Daria para apenas estilizar o botão diretamente e no componente não usar o ButtonContainer já que ele não tem nenhum estilo.
```suggestion
export const ButtonContainer = styled.button`
  ...estilo do btn
`
```